index.js

```diff
@@ -134,8 +135,9 @@ const createCdrCountQuery = ({account_sid, trunk, direction, answered, days, sta
   return sql;
 };
 
-const createAlertsQuery = ({account_sid, alert_type, page, page_size, days, start, end}) => {
+const createAlertsQuery = ({account_sid, target_sid, alert_type, page, page_size, days, start, end}) => {
```

Comment:
Hmm, not sure about this.  First of all, on the Alerts page I would like to show all alerts for the account.  So we need to query for account and get all of the alerts.  Second, call_sid is a field, not a tag (it cant be a tag because there are too many possible values), and I am not sure you can query by a field (?)

Reply:
Yes, I agree, moved target_sid to fields